containerfiles/unstable

```diff
@@ -0,0 +1,150 @@
+FROM quay.io/fedora/fedora-bootc@sha256:b11b87e0f1d843c14e7458b5d12eb70682c9a2a63eb46b6592bceeddd1a40162
+
+ARG kver=6.12.10-203
+
+# Force dnf5
+RUN rm -f /usr/bin/dnf && ln -s /usr/bin/dnf5 /usr/bin/dnf
```

Comment:
There are still a few minor features missing from DNF 5. The most useful one I've come to use in DNF 3 and 4 is this one to easily and quickly add a remote repository: `dnf config-manager --add-repo ${REPO_URL}`.

I can understand the confusion from an end-user perspective, though, if they try to use DNF 4 and it errors out instantly saying it's not compatible with OSTree systems.

Reply:
Now I can't remember what it was exactly, but I had to do this early in the build because there was a step that was failing because the wrong dnf was being called.